src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -46,14 +47,24 @@ class MovingWindow:
     [`OrderedRingBuffer`][frequenz.sdk.timeseries._ringbuffer.OrderedRingBuffer]
     documentation.
 
+    Resampling might be required when working with streams of samples from
+    different sources at different sampling rates or periods, and it can be
+    set by specifying the resampler config parameter so that the user can
+    control the granularity of the samples to be stored in the underlying
+    buffer.
+
+    If resampling is not required, the resampler config parameter can be
+    set to None or the resampling period can be set equal to the input
+    sampling period in which case the MovingWindow will not perform any
+    resampling.
 
     **Example1** (calculating the mean of a time interval):
 
     ```
     window = MovingWindow(
         size=timedelta(minutes=5),
         resampled_data_recv=resampled_data_recv,
-        sampling_period=timedelta(seconds=1)
+        input_sampling_period=timedelta(seconds=1)
```

Comment:
Maybe mention this change in name (breaking change) explicitly in the release notes?

Reply:
done